### ISSUE TITLE
"non project file access" warning when 2 projects are opened

### ISSUE THREAD

**Issue by bgranvea** [krasa/EclipseCodeFormatter#95]
I have two projects opened in two Intellij Idea windows. When I modify a file in the second project and the formatter tries to format the file, I have an Intellij warning:

"Non-project file access
These files do not belong to the project."

This is very annoying and didn't happen with previous version of the plugin.

**Comment by krasa** [comment]
and do those files belong to the project? could you make a screeenshot?

**Comment by bgranvea** [comment]
Yes the files belong to the project. Here is a screenshot. 

When I format the file in the 2nd project (below), the warning appears. It seems to appear in the 1st project as if the eclipse formatter plugin is running only in first project and is not allowed to access files in 2nd project.

![image](https://cloud.githubusercontent.com/assets/1116217/15244246/7abd4950-1900-11e6-9c7e-cfd003d4cbcf.png)

**Comment by krasa** [comment]
Cannot reproduce, and this is even stranger than the other bug. 
Does it happen when you switch to normal formatter? (ctrl+shift+A - "Switch Code Formatter") 
And does it happen even when you format without import optimizing? Because the only real change in the last version was in imports...

**Comment by krasa** [comment]
Also you might like this plugin https://plugins.jetbrains.com/plugin/7635?pr=

**Comment by bgranvea** [comment]
it seems to be an interaction with the "save actions" plugin https://plugins.jetbrains.com/plugin/7642 which was also updated a few days ago:

- if I put default formatter in both projects, I still got the "non project file access" error when saving a file (I've configured save actions with organize imports, reformat file and reformat only changed code)
- even with a single opened project, when I save a file, there are multiple "XXX formatted successfully" message for the same file. Very strange.

I try to downgrade the other plugin to see what happens. BTW, this could explain the other bug with "unshelve changes".
